### DISCUSSION TITLE
Greatest challenges learning Miryoku and tips on how to overcome

### DISCUSSION THREAD

**Discussion by abandon42** [manna-harbour/miryoku#110] [Q&A] [answered]
Hello everyone,

Hoping to collect some learning tips from everyone.

My two largest challenges were:

1. Not having a clue about home row mods (easily fixed)
 
2. Constantly hitting Enter instead of backspace. 
Since I'm new to my thumb having to do anything remotely complex, and cannot count the number of typos I've sent to colleagues while intending to correct them. 

After seeing similar experiences, I finally used a new keycap profile to create homing. I suspect this will be super helpful to anyone learning miryoku, particularly if new to staggered kbs. 

I wonder if having corrections next to sends is optimal from an interface perspective. I think we're trying to even out thumb usage left to right, but it may make sense to have more cognitive distance or effort like regular ANSI has. While the keys are often adjacent, they require a lot more hand movement to get the pinky to reach, which helps to create more cognitive distinction.
I'll keep at this for now with the hope that homing thumb keys will address it.

**Comment by agisga** [comment]
Re 2. I had a similar issue also because thumb clusters differ somewhat on different keyboards that I use (with respect to the position and which key is most comfortable to hit). So, now I use Enter only from the nav layer, and Tab only from the fun layer (and alternatively as a home row combo). On base layer Tab is replaced by another Spc for example.

**Comment by manna-harbour** [comment, marked as answer]
For 1. see https://precondition.github.io/home-row-mods. Also, I'll be adding a section to the docs, and adding some options to the implementations.

For 2. if you always home on the home thumb I don't think this should be a problem with more practice, as you should be able to just feel when your thumb is in a different position. If you switch between different keyboards, homing keys on the home thumbs as you mentioned can be useful.

Enter is with backspace and delete for editing. But if you're having trouble with enter on Base, @agisga's idea of disabling it and only using enter on Nav is a good one for while you are getting used to the thumbs. And even after, it might be a good location for one shot shift.
